### PR TITLE
Add services setup service

### PR DESCRIPTION
**Pull request by pixelblend**
A service that manages services!

Building on the [conditional service starting][1], this process
manipulates that conditionality, and puts a UI on top of the system.

Once changes have been saved, your pi will require a reboot.

The next stage will be to manage which apps are running on the system.

[1]:https://github.com/andrewn/neue-radio/pull/26

### CODE REVIEW

File: deployment/systemd/setup.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Radioan Service Setup
```
Comment (andrewn) [inline deployment/systemd/setup.service]: "Setup" makes me think of something that only runs once and not a system service. Perhaps Config is more appropriate?

File: deployment/systemd/setup.service [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+[Service]
+WorkingDirectory=/opt/radiodan/rde/services/setup
+ExecStart=/usr/local/bin/npm start
+Restart=on-failure
+StandardOutput=syslog
+SyslogIdentifier=rde-debug
+Type=idle
+Environment=PORT=5020
```
Comment (andrewn) [inline deployment/systemd/setup.service]: Is it time to have a policy for port numbering? We currently have `5000`, `5001`, `5002`, `5020`, `5100`, `8080` 😭 .

**Comment by andrewn** [comment]
This is cool! I left a couple of more general comments about how we might organise port numbering. And also, although having lots of tiny services `setup`, `debug` is 👍 for development, do you think it might start getting a bit confusing to remember them all as a user?

File: deployment/systemd/setup.service (hunk 1 above)
Comment (pixelblend) [reply deployment/systemd/setup.service]: I kept going back and forth between them. Maybe there's a better name than both of them! I'll have a think.

File: deployment/systemd/setup.service (hunk 2 above)
Comment (pixelblend) [reply deployment/systemd/setup.service]: It's nuts, and needs fixing. I have some ideas around setting Environment Variables from external files, which might help with this.

Comment (pixelblend) [reply deployment/systemd/setup.service]: To be clear, I'll set it up in another PR that can be merged before this.

**Comment by pixelblend** [comment]
Should we combine the services into some kind of meta-service?

**Comment by andrewn** [comment]
>> do you think it might start getting a bit confusing to remember them all as a user?

> Should we combine the services into some kind of meta-service?

I think we should. Or, we could have some kind of index service that lists them and links to them on their individual ports. We could use the [Web App Manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest) JSON format to specify `service` and `app` info in a consistent way.

**Comment by pixelblend** [comment]
:+1: small steps are best

**Pull request merged**